packages/databricks-vscode/src/ui/bundle-resource-explorer/PipelineRunStatusTreeNode.ts

```diff
@@ -0,0 +1,135 @@
+import {
+    BundleResourceExplorerTreeItem,
+    BundleResourceExplorerTreeNode,
+} from "./types";
+import {TreeItemCollapsibleState} from "vscode";
+import {ContextUtils, RunStateUtils} from "./utils";
+import {SimplifiedRunState, sentenceCase} from "./utils/RunStateUtils";
+import {GetUpdateResponse} from "@databricks/databricks-sdk/dist/apis/pipelines";
+import {PipelineRunStatus} from "../../bundle/run/PipelineRunStatus";
+import {TreeItemTreeNode} from "./TreeItemTreeNode";
+import {ConnectionManager} from "../../configuration/ConnectionManager";
+
+function getSimplifiedUpdateState(
+    update?: GetUpdateResponse["update"]
+): SimplifiedRunState {
+    if (update?.state === undefined) {
+        return "Unknown";
+    }
+
+    switch (update.state) {
```

Comment:
We could either add a default case here with the unknown state, or a default case to the getThemeIconForStatus, with question mark icon